libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -45,6 +48,7 @@ export class BookSearchComponent implements OnInit {
 
   addBookToReadingList(book: Book) {
     this.store.dispatch(addToReadingList({ book }));
+    this.openSnackBar("Book added", "Undo", book);
```

Comment:
Is this the right place to do this? What if the API call fails?

Reply:
updated